src/c++/perf_analyzer/perf_analyzer.cc

```diff
@@ -863,9 +863,8 @@ PerfAnalyzer::Run(int argc, char** argv)
 
   // C Api backend required info
   const std::string DEFAULT_MEMORY_TYPE = "system";
-  std::string triton_server_path;
+  std::string triton_server_path = "/opt/tritonserver";
```

Comment:
Nice addition, makes use of C API a bit less verbose for the average case :slightly_smiling_face:

Reply:
Thanks! Yeah, it was annoying to specify this each time :)